oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -565,31 +683,6 @@ int DumpHandler::fileAckWithMetaData(uint32_t metaDataValue1,
                     << e.what() << "\n";
             }
         }
-        // commented as the DBUS support is not yet in
-        /* else
-         {
-             DBusMapping dbusMapping;
-             dbusMapping.objectPath =
-                 "/xyz/openbmc_project/dump/resource/entry/";
-             dbusMapping.interface = "com.ibm.Dump.Entry.Resource";
-             dbusMapping.propertyName = "requestToken";
-             dbusMapping.propertyType = "uint32";
-
-             pldm::utils::PropertyValue value =
-                 "com.ibm.Dump.Entry.Resource.requestToken";
-
-             try
-             {
-                 pldm::utils::DBusHandler().setDbusProperty(dbusMapping,
-                                                            metaDataValue1);
-             }
-             catch (const std::exception& e)
-             {
-                 std::cerr << "failed to set token for resource dump, ERROR="
-                           << e.what() << "\n";
-                 return PLDM_ERROR;
-             }
-         }*/
 
         if (fs::exists(resDumpDirPath))
```

Comment:
resDumpDirPath always points to file name 1. Change that to be dynamic according to file handle